specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/mfe.json

```diff
@@ -14420,6 +14420,11 @@
             "read"
           ]
         },
+        "pathOnCompute": {
+          "description": "Input Asset Delivery Path.",
+          "type": "string",
```

Comment:
>"type": "string", [](http://example.com/codeflow?start=10&length=17)

Is this supposed to be an URI path?
If so, consider defining validation regex to restrict the valid set of inputs, to make it clear if people are allowed to send absolute URIs, what protocol etc,